src/js/components/SharedComponents/ProgressComponent.jsx

```diff
@@ -26,11 +29,12 @@ export default class Progress extends React.Component {
         const progressBar = [];
         const progressLabels = [];
         const stepNames = this.props.stepNames;
+        const stepLink = this.props.stepLink;
 
         for (let i = 1; i <= this.props.totalSteps; i++) {
             if (i < this.props.currentStep) {
                 progressBar.push(<li key={i} className="usa-da-progress-bar-step-done"><span className="step">{i}</span></li>);
-                progressLabels.push(<li key={i} className="usa-da-progress-bar-step-done"><span className="name">{stepNames[i - 1]}</span></li>);
+                progressLabels.push(<li key={i} className="usa-da-progress-bar-step-done"><a href={stepLink[i-1] +"/"+ this.props.id} ><span className="name">{stepNames[i - 1]}</span></a></li>);
```

Comment:
Make the link only the words (with maybe a little bit of padding, like a few pixels). Right now as long as you're on the right y-coordinate within the div you can click on the link, which seems misleading especially since it's not super obvious right away that it's a link.

It would also be nice to be able to click anywhere in the circle for the same effect.